notebooks/thunder_as_torch_compile_backend_tutorial.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<p>Maybe this section could be called "Hello World" and be the second cell in the notebook. </p><p> </p><p>Let's avoid words like "simply" and "seamlessly." Just focus on explaining, without editorializing, what's being shown. For example, "Here's a simple example of using ThunderFX to run a PyTorch program:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>